### PR TITLE
Added scraper for Faulkner Co, AR

### PR DESCRIPTION
**Pull request by mcoberley**
Added scraper for Faulkner County, AR police department. Included several samples of the data. The actual data itself is a lot bigger than what's provided. Tried to make the README as helpful as possible.

### CODE REVIEW

**Comment by josh-chamberlain** [comment]
Looks like this is well-documented and obeys the `robots.txt`, anything I'm missing before approval @CaptainStabs ?

**Review by CaptainStabs — APPROVED** [review]
Looks good!

**Pull request merged**
